client/packages/requisitions/src/RequestRequisition/DetailView/Footer/StatusChangeButton.tsx

```diff
@@ -110,15 +132,21 @@ const useStatusChangeButton = () => {
 export const StatusChangeButton = () => {
   const { selectedOption, getConfirmation } = useStatusChangeButton();
   const isDisabled = useRequest.utils.isDisabled();
+  const { userHasPermission } = useAuthContext();
 
   if (!selectedOption) return null;
   if (isDisabled) return null;
 
+  const hasPermission =
+    selectedOption.value === RequisitionNodeStatus.Sent
+      ? userHasPermission(UserPermission.RequisitionSend)
+      : true;
+
   return (
     <ButtonWithIcon
       color="secondary"
       variant="contained"
-      disabled={isDisabled}
+      disabled={!hasPermission}
```

Comment:
hmm, I do like the idea of disabling thing rather then completely hiding them, but I do think that we also need to indicate why it's disabled, is that possible to do ?

Reply:
Nice! yes, I wondered about that.
have added a tooltip around the button, if the user doesn't have permission: 
<img width="233" alt="image" src="https://github.com/openmsupply/open-msupply/assets/9192912/1266a843-e7c4-4d81-a805-2763c44abb27">

Reply:
🙏